source/iface/half-prec-math-func.rst

```diff
@@ -7,3 +7,422 @@
 *****************************
 Half precision math functions
 *****************************
+
+In SYCL the half precision math functions are defined in the namespace
+``sycl::half_precision``.
+The functions that are available within this namespace are specified
+in the list below.
+These functions are implemented with a minimum of 10-bits of accuracy
+i.e. the maximum error is less than or equal to ``8192 ulp``.
+
+``cos``
+=======
+
+::
+
+  float cos(float x)                (1)
+
+  template<typename NonScalar>      (2)
+  /*return-type*/ cos(NonScalar x)
+
+Overload (1):
+
+Preconditions: The value of ``x`` must be in the range [-216, +216].
+
+Returns: The cosine of ``x``.
+
+Overload (2):
+
```

Comment:
Same comment as previous PR.  Can we unify the overload heading style with the other pages in this chapter?  It's possible that more that just this page need to be updated, but we can discuss if needed.

Reply:
Fixed with https://github.com/KhronosGroup/SYCL_Reference/pull/168/commits/0d027673f5299b69320e8ce40d60df334864f5e9